src/javascript/components/TodoApp.js

```diff
@@ -0,0 +1,125 @@
+import TodoItemModel from "./model/TodoItemModel.js";
+import TodoInput from "./TodoInput.js";
+import TodoList from "./TodoList.js";
+import TodoCount from "./TodoCount.js";
+import TodoFilter from "./TodoFilter.js";
+
+function TodoApp() {
+  const initialData = JSON.parse(localStorage.getItem("listData"));
+
+  let id = initialData ? initialData.length : 0;
+  this.todoItems = initialData ? initialData : [];
+
+  const saveData = () => {
+    localStorage.setItem("listData", JSON.stringify(this.todoItems));
+  };
+
+  const countTarget = document.querySelector(".todo-count strong");
+  const todoCount = new TodoCount(countTarget);
+
+  const todoList = new TodoList({
```

Comment:
todoList 를 새로 만드는 부분에서 너무 길어지는 것 같은데요,
가독성을 위해서 각각 function을 따로 만들어 주는 건 어떨까요?
```suggestion
  const todoList = new TodoList(this.deleteItem(id), this.completeItem(id), this.editItem(Id))
  const deleteItem = (id) => {...}
```